docs/Concepts/Privacy/Flexible-PrivacyGroups.md

```diff
@@ -60,4 +60,44 @@ group ID and passes the ID to Besu when creating a privacy group.
     participants. If you create a privacy group with an existing privacy group ID, the existing
     privacy group is overwritten.
 
-    To ensure unique privacy group IDs, using 256-bit SecureRandom is recommended.
+    To ensure unique privacy group IDs, we recommend using 256-bit SecureRandom.
+
+## Multi-tenancy
+
+When using [multi-tenancy](Multi-Tenancy.md) with flexible privacy groups, each user provides a JSON Web Token (JWT)
+which allows Besu to check that the user has access to functionality and data associated with a privacy group.
+
+Using multi-tenancy with flexible privacy groups is more complex than with [offchain privacy groups](Privacy-Groups.md)
+because users may try to access a privacy group they were once a member of but later removed from.
+
+When multi-tenancy is enabled and a user requests access to a privacy group they were once a member of but later
+removed from, Besu allows the user access to the following functionality and data associated with the privacy group:
+
+- Private transactions using `priv_getTransaction` and private transaction receipts using
+  [`priv_getTransactionReceipt`](../../Reference/API-Methods.md#priv_gettransactionreceipt) from blocks up to (and
+  including) the block containing the [privacy marker transaction (PMT)](Private-Transaction-Processing.md) that removed
+  the user.
+  
+    !!! note
+
+        A removed group member may have access to some private transactions in the same block after the removal PMT.
+        There may be more private transactions sent to that privacy group version in later blocks, but all of these
```

Comment:
do we need to explain the version a bit more?

Reply:
There's currently no documentation on privacy group version. Is it a flexible privacy group thing or does it also apply to offchain privacy groups?